auth_oauth_environment/models/auth_oauth_provider.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import json
+from lxml import etree
+
+from odoo import fields, models, api, _
+from odoo.addons.server_environment import serv_config
+
+
+class AuthOAuthProvider(models.Model):
+
+    _inherit = 'auth.oauth.provider'
+
+    managed_by_env = fields.Boolean(compute='_compute_server_env')
+    provider_simple_name = fields.Char(compute='_compute_simple_name')
+    client_id = fields.Char(string='Client ID', compute='_compute_server_env')
+    enabled = fields.Boolean(string='Allowed', compute='_compute_server_env',
+                             search='_search_enabled')
+
+    def _compute_server_env(self):
+        for provider in self:
+            provider_section_name = provider._get_provider_section_name()
```

Comment:
Here you currently receive a list, see comment on method `_get_provider_section_name`

Reply:
@yvaucher @grindtildeath  is this still relevant?